PROTOCOL.md

```diff
@@ -1988,12 +2038,13 @@ Query Parameters | **startingVersion** (type: Int64, optional): The starting ver
 <td>Body</td>
 <td>
 
-A sequence of JSON strings delimited by newline. Each line is a JSON object defined in [Table Metadata Format](#table-metadata-format).
+A sequence of JSON strings delimited by newline. Each line is a JSON object defined in [API Response Format](#api-response-format).
 
 The response contains multiple lines:
 - The first line is [a JSON wrapper object](#json-wrapper-object-in-each-line) containing the table [Protocol](#protocol) object.
 - The second line is [a JSON wrapper object](#json-wrapper-object-in-each-line) containing the table [Metadata](#metadata) object.
-- The rest of the lines are [JSON wrapper objects](#json-wrapper-object-in-each-line) for [CDF files](#cdf-files) of the change data feed.
+- The rest of the lines are [JSON wrapper objects](#json-wrapper-object-in-each-line) for [Data Change Files](#data-change-files) of the change data feed.
+- Historical [Metadata](#metadata) will be returned if includeHistoricalMetadata is set to true.
```

Comment:
Nested bullet point? Also include that ordering does not matter?

Reply:
added in both queryTable and queryTableChanges.